doc/doxygen/src/ForHosts.dox

```diff
@@ -98,21 +98,32 @@
  *   user with an additional manual configuration mechanism for your
  *   specific API session if you so wish.
  *
- * - Create, persist and configure a @ref Context as appropriate for
- *   all calls to the API.
+ * - Create (and configure) a @ref Context as appropriate for
+ *   all calls to the API. The Context instance should be kept alive
+ *   and re-used for API calls that are related to the same user
+ *   session, or logical set of interactions. For example, one per
+ *   document. Use @fqref{hostApi.Manager.createChildContext}
+ *   "createChildContext" to split off a new Context should you need to
+ *   re-configure it for any specific series of actions, whilst
+ *   maintaining the association.
  *
  * - Ask the @ref manager if they support resolve for the relevant
  *   data by supplying its @ref trait_set to
  *   @fqref{hostApi.Manager.managementPolicy} "Manager.managementPolicy"
  *   with a @ref Context with `kRead` access, respecting the returned
  *   traits - this may be a subset if the manager is only capable of
- *   resolving some of an entity's traits.
+ *   resolving some of an entity's traits. Note that policy is
+ *   runtime invariant for any given context/access/trait set.
  *
  * - Always check any suspected entity references with
  *   @fqref{hostApi.Manager.isEntityReferenceString}
  *   "Manager.isEntityReferenceString" before passing to any other API
  *   calls.
  *
+ * - If you support distributed processing provide a mechanism to
+ *   @fqref{hostApi.Manager.persistenceTokenForContext} "persist" a @ref
+ *   Context across processes.
```

Comment:
[Question] Curious where this comes from? Seems like a non-simple thing to call out in the requirements for simple resolution section.

Reply:
Rendering different frames across different machines is very common in post production. This mechanism is essential to allow any kind of temporal stability.